plugin-manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "api": {
+    "class_name": "APBabyApplicasterPlayer",
+    "modules": [
+      "ZappPlayerPluginBaby",
+      "ZappPlayerPlugins"
+    ]
+  },
+  "author_name": "Alex Zchut",
+  "author_email": "a.zchut@applicaster.com",
+  "manifest_version": "5.8.0",
+  "name": "APBabyPlayer",
```

Comment:
you have to use the same name from before

Reply:
i did it.  same name and identifier from previous version